### DISCUSSION TITLE
Immich looks great! Does it work with external storage?

### DISCUSSION THREAD

**Discussion by notflip** [immich-app/immich#2944] [Q&A] [answered]
I've got a Hetzner server 2CPU 4GB RAM, but the storage is low, would love to run Immich on there.
Is it possible to use external cheap storage such as B2?

**Comment by rjhenry** [comment, marked as answer]
As [B2 can be mounted via rclone](https://rclone.org/b2/), it should be able to be. See [this discussion](https://github.com/immich-app/immich/discussions/1683) and [this PR](https://github.com/immich-app/immich/pull/2832).

Note I haven't tried this yet!
